crates/config/src/constants.rs

```diff
@@ -5,7 +5,7 @@ pub const MAX_WITHDRAWAL_SIZE: usize = 50_000;
 // 25 KB
 pub const MAX_WRITE_DATA_BYTES_LIMIT: usize = 25 * 1024;
 // 2MB
-pub const MAX_READ_DATA_BYTES_LIMIT: usize = 1024 * 1024 * 2;
+pub const MAX_TOTAL_READ_DATA_BYTES_LIMIT: usize = 1024 * 1024 * 2;
```

Comment:
`MAX_TOTAL_READ_DATA_BYTES` or `TOTAL_READ_DATA_BYTES_LIMIT`?

Reply:
updated, rename to `MAX_TOTAL_READ_DATA_BYTES`